internal/cli/cmd/dac/build/build.go

```diff
@@ -124,7 +123,7 @@ func (o *option) processFile(file string, extension string) error {
 		// we must be in the closest directory to the file.
 		folder := filepath.Dir(file)
 		extractedFile := filepath.Base(file)
-		cmd = exec.Command("go", "run", extractedFile, "--output", o.Output) // #nosec
+		cmd = exec.Command("go", "run", extractedFile, "--output", o.Output, strings.Join(o.args, " ")) // #nosec
```

Comment:
probably you can add the same changes for cuelang ?

Reply:
Ups, missed this.